commands/Misc/hug.js

```diff
@@ -0,0 +1,27 @@
+const { Command } = require('klasa');
+const fetch = require('node-fetch');
+const { MessageEmbed } = require('discord.js');
+
+module.exports = class extends Command {
+
+    constructor(...args) {
+        super(...args, {
+            cooldown: 5,
+            aliases: [],
+            usage: '<user:username>',
+            description: 'Hugs the mentioned user and sends an image in chat.',
+            extendedHelp: ''
+        });
+    }
+
+    async run(message, [user]) {
+        await fetch('https://nekos.life/api/v2/img/cuddle').then(res => res.json()).then(body => {
```

Comment:
this needs to be rewritten a bit please. Try to avoid using .then() too much its confusing. assign the json result to a variable then deconstruct that variable then u can use it how u like. It will look a lot nicer